README.md

```diff
@@ -0,0 +1,171 @@
+<a name="readme-top"></a>
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Thanks for the idea about the logo. I actually thought of it some time ago and produced something which I think still needs lots of work.